config/locales/en.yml

```diff
@@ -82,7 +82,7 @@
       responsible_body: Orders will be placed centrally
     privacy_notice: Privacy notice
     school_home: Get devices for your school
-    school_cannot_order_devices: You cannot order devices yet
+    school_cannot_order_devices: Your school cannot order devices yet
```

Comment:
We need both titles, one for order users, and one for non order users:

See these two:
https://ghwt-design-history.herokuapp.com/ordering-devices-school/#you-cannot-order-devices-yet
https://ghwt-design-history.herokuapp.com/ordering-devices-school/#your-school-cannot-order-devices-yet-non-order-user

Reply:
Thanks, should be sorted now